### DISCUSSION TITLE
博客配置出错

### DISCUSSION THREAD

**Discussion by GianYue** [Qexo/Qexo#497] [Q&A] [answered]
在qexo博客配置界面提示如下图，我已将repo转为hexo源库，请问该怎么办，小白真心求教
![QQ图片20240130213613](https://github.com/Qexo/Qexo/assets/84278589/114c1a7e-2fa1-4540-9e68-530ea30cacf9)
![QQ图片20240130214511](https://github.com/Qexo/Qexo/assets/84278589/972565fc-6ad9-434b-97a9-b214fc1b8cae)
![QQ图片20240130214554](https://github.com/Qexo/Qexo/assets/84278589/8c15a0d0-247e-4746-a899-b4446f954e54)

**Comment by am-abudu** [comment, marked as answer]
强制提交即可
